src/components/place-card/place-card.tsx

```diff
@@ -0,0 +1,97 @@
+import clsx from 'clsx';
+
+type PlaceCard = {
+  // id: string;
+  title: string;
+  type: string;
+  price: number;
+  // city: {
+  //   name: string;
+  //   location: {
+  //     latitude: number;
+  //     longitude: number;
+  //     zoom: number;
+  //   };
+  // };
+  // location: {
+  //   latitude: number;
+  //   longitude: number;
+  //   zoom: number;
+  // };
+  isFavorite: boolean;
+  isPremium: boolean;
+  rating: number;
+  previewImage: string;
+};
+
+export default function PlaceCard(props: PlaceCard): JSX.Element {
+  const { title, type, price, isFavorite, isPremium, rating, previewImage } =
+    props;
+
+  const bookmarkClass = clsx(
+    isFavorite && 'place-card__bookmark-button--active'
+  );
```

Comment:
лучше сразу объединить с статическими классами
```suggestion
  const bookmarkClass = clsx('place-card__bookmark-button button',
    isFavorite && 'place-card__bookmark-button--active'
  );
```